last.fm/chart.js

```diff
@@ -5,26 +5,28 @@ function Chart(api_key) {
 Chart.prototype = Object.create(Loader.prototype);
 Chart.prototype.constructor = Chart;
 
+Chart.prototype.getTopArtistsHandler = function() {
+  loader.request.onreadystatechange = function() {
```

Comment:
you don't need `loader.` if you use prototype
are you sure you understand how inheritance works in JS?